src/hooks/configs.ts

```diff
@@ -0,0 +1,130 @@
+import {
+  ResourceTypeEnum,
+  LearningResourcesSearchRetrieveDepartmentEnum as DepartmentEnum,
+  LearningResourcesSearchRetrieveLevelEnum as LevelEnum,
+  LearningResourcesSearchRetrievePlatformEnum as PlatformEnum,
+  LearningResourcesSearchRetrieveOfferedByEnum as OfferedByEnum,
+  LearningResourcesSearchRetrieveSortbyEnum as SortByEnum,
+  // ContentFile Search Enums
+  ContentFileSearchRetrieveSortbyEnum as ContentFileSortEnum
+} from "../open_api_generated"
+
+type Endpoint = "resources" | "content_files"
+
+type SearchParams = {
+  endpoint: Endpoint
+  activeFacets: {
```

Comment:
Can we reuse `Facets` from `facet_display/types.ts` aside from the booleans (though these are currently missing the generated API enums)?